packages/bot-web-ui/jest.config.js

```diff
@@ -12,5 +12,16 @@ module.exports = {
         '^Stores/(.*)$': '<rootDir>/src/stores/$1',
         '^Utils/(.*)$': '<rootDir>/src/utils/$1',
     },
+    coveragePathIgnorePatterns: [
+        '<rootDir>/.eslintrc.js',
+        '<rootDir>/jest.config.js',
+        '<rootDir>/coverage/lcov-report',
+        '<rootDir>/dist',
+        '<rootDir>/build',
+        '<rootDir>/docs',
+    ],
+    globals: {
+        __webpack_public_path__: '',
```

Comment:
@rupato-deriv are you sure we need it without path? 
@ali-hosseini-deriv could you please take a look?

Reply:
yes this was required because when we try to mock the dashboard component the webpack path goes undefined so i set it to default path now here 
![Uploading Screenshot 2023-04-04 at 11.18.52 AM.png…]()